### PR TITLE
make a list of contributors

### LINKED ISSUE #211: add AUTHORS, CONTRIBUTORS or CREDITS to toplevel dir
it's useful so people know who worked on the software.
also it can be referred to in (c) notices (if appropriate).

.rst maybe?

### PR DESCRIPTION
**Pull request by SwartzCr**
fixes #211 
made by running `git shortlog --summary --email` and then doing some cleanup
I optimized for non-gmail address and instead opted for work or personal website emails. That said it still probably has a lot of addresses in it that people might not like being posted publicly (even though they are already in the git history...)

### CODE REVIEW

**Comment by ian-kelling** [comment]
I do not like the  "Authors / Contributors" title, because many contributors did not commit code: they filed bugs, reviewed code, designed, wrote widely read documentation not included in the repo, etc. and free software communities have a tendency to (not sure the best wording) marginalize / not recognize those contributions/contributors, and this does that by implying that the people not on this list are not contributors. I'd at least change the title to "Commiters", and at most, something like "Commiters. Look online to find the many valuable contributors who have not commited code."

**Comment by SwartzCr** [comment]
Good point @ian-kelling. If you can think of a way to find a list of people who contributed outside of committing code, I'd be happy to include them

**Comment by ian-kelling** [comment]
> If you can think of a way to find a list of people who contributed outside of committing code, I'd be happy to include them

The information is almost entirely on the public internet, so clearly there is a way to find it, but I doubt anyone wants to do that work, so I think it's probably better simply to tweak the title as I suggested.

**Comment by SwartzCr** [comment]
I'm ... going through and making github links for all the ones I can find

https://www.youtube.com/watch?v=U24OvWVdVwA#?t=1m20s

**Comment by bmw** [comment]
That sounds...painful. Again, feel free to ignore my suggestion and just leave the e-mail address if you want. I only said it because we'll be putting people's e-mail addresses in an easily scrapeable file, but it's already in our git history and plenty of other repos do it.

**Comment by SwartzCr** [comment]
done! check it out!!
@bmw now there should be no more emails, ready to eview and merge

File: docs/contributors.rst [hunk 1]
```diff
@@ -0,0 +1,272 @@
+Authors / Contributors
```
Comment (zjs) [inline docs/contributors.rst]: This file should be removed, right?

**Comment by SwartzCr** [comment]
@zjs should be fixed now

File: AUTHORS.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+* [Dominic Cleal](https://github.com/domcleal)
+* [Dominic Lüchinger](https://github.com/dol)
+* [Douglas José](https://github.com/douglasjose)
+* [Eric Rescorla](https://github.com/ekr)
+* [Eric Engestrom](https://github.com/1ace)
+* [Eric Wustrow](https://github.com/ewust)
+* [Erica Portnoy](https://github.com/ohemorange)
+* [Erik Rose](https://github.com/erikrose)
+* [Eugene Kazakov](https://github.com/xgin)
+* [Giovanni Pellerano](https://github.com/evilaliv3)
```
Comment (bmw) [inline AUTHORS.md]: Not to nit this point too much, but the sorted order still seems off. This line is one example.

**Pull request merged**